src/styles/blocks/header.scss

```diff
@@ -0,0 +1,78 @@
+@use '../utils/variables' as *;
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  font-weight: 500;
+
+  &__logo {
+    align-self: center;
+    display: flex;
+  }
+}
+
+.navigation {
+  display: flex;
+  align-items: center;
+}
+
+.nav {
+  &_list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding-left: 0;
+  }
+
+  &_item {
+    display: flex;
+  }
+
+  &_link {
+    color: $black-color;
+    text-decoration: none;
+    font-size: 12px;
+    text-transform: uppercase;
+    align-items: center;
+    display: flex;
+    height: 60px;
+    margin-left: 20px;
+
+    &:visited {
+      color: $black-color;
+    }
+
+    &:hover {
+      color: $blue-color;
+    }
+
+    &:active {
+      color: $blue-color;
+      position: relative;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+  text-decoration: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  align-items: center;
+  display: flex;
+  height: 60px;
+  margin-left: 20px;
+  color: $blue-color;
```

Comment:
The `.is-active` class duplicates many styles from the `&_link` class. Consider using a modifier class or extending the `&_link` class to avoid redundancy and ensure consistency.